pkgs/servers/http/tengine/default.nix

```diff
@@ -54,15 +60,30 @@ stdenv.mkDerivation rec {
     "--without-mail_pop3_module"
     "--without-mail_imap_module"
     "--without-mail_smtp_module"
-  ] ++ optional withIPv6 [
-    "--with-ipv6"
```

Comment:
I guess IPv6 support is now built-in by default?

Reply:
yes, ipv6 by default